puppet/modules/chassis/manifests/site.pp

```diff
@@ -21,6 +21,10 @@
 		content => template('chassis/site.nginx.conf.erb'),
 		notify  => Service['nginx']
 	}
+	file { "/etc/nginx/sites-available/${name}.d": 
```

Comment:
As a temporary fix for compatibility, can we wrap this in a `if ( ! defined( File["/etc/nginx/sites-available/${name}.d"] ) ) {` check?

Reply:
can use that in my extension too